### DISCUSSION TITLE
[GUIDE NEEDED] How to get wallet provider from Appkit v5.1.5

### DISCUSSION THREAD

**Discussion by ms-safiq** [WalletConnect/walletconnect-monorepo#5325] [Developer Support]
Hi,

I recently migrated web3modal/ethers from v5.0.11 to v5.1.5. Seems like there were some method removed in the latest version. I was using modal.subscribeProvider(...) or modal.getWalletProvider() and these are removed from the latest version. 

I am unable the find the details in the doc as well. The appkit documentation is not updated. 

Please help me with how to get the walletprovider from the connected wallet in the appkit v5.1.5?

Thanks
Mohammed Safiq SA

**Comment by oyin25** [comment]
Have you found solutions, please i need help as well.

**Reply by arein** [reply]
Can you please upgrade to latest? https://docs.reown.com/appkit/upgrade/from-w3m-to-reown
